src/components/menu-item/menu-item.scss

```diff
@@ -16,6 +15,10 @@
   @apply w-full;
 }
 
+:host(:not([layout="vertical"])) {
+  @apply h-full;
+}
```

Comment:
This change is related to visual regression found while debugging navigation component.